### PR TITLE
feat: make status tags on design guidelines accessible

### PR DESCRIPTION
**Pull request by katieporter**
Remove background color from tags on design guidelines so they meet accessibility reqs.

### CODE REVIEW

**Comment by SiTaggart** [comment]
This now might suffer from not being able to distinguish the difference from the plain text that surrounds it. 

If I look at it on my phone, it's just an icon with some text. Nothing really stands out to say that it's interactive in its own right.

**Comment by katieporter** [comment]
@SiTaggart updated based on our convo!

File: packages/paste-website/src/pages/getting-started/design/index.mdx [hunk 1]
```diff
@@ -11,13 +11,14 @@ import {Text} from '@twilio-paste/text';
...
 import {Tooltip} from '@twilio-paste/tooltip';
+import {Popover, PopoverContainer, PopoverButton} from '@twilio-paste/popover';
 import {Stack} from '@twilio-paste/stack';
 import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
 
 import { SuccessIcon } from "@twilio-paste/icons/esm/SuccessIcon";
 import { WarningIcon } from "@twilio-paste/icons/esm/WarningIcon";
 import { ErrorIcon } from "@twilio-paste/icons/esm/ErrorIcon";
-
+import { InformationIcon } from "@twilio-paste/icons/esm/InformationIcon";
```
Comment (SiTaggart) [inline packages/paste-website/src/pages/getting-started/design/index.mdx:21]: Just need to delete these two unused imports and you're good

**Comment by katieporter** [comment]
@SiTaggart good catch - fixed!

**Review by SiTaggart — APPROVED** [review]

**Pull request merged**